extendr-api/src/lib.rs

```diff
@@ -127,6 +130,20 @@ pub unsafe fn register_call_methods(info: *mut libR_sys::DllInfo, methods: &[Cal
 //     Robj::eval_string(rcode.as_str()).unwrap();
 // }
 
+pub fn print_r_output<T: Into<Vec<u8>>>(s: T) {
+    let cs = CString::new(s).expect("NulError");
```

Comment:
Did you mean to write "NullError"? (two l's) See also line 141.

Reply:
Intentional, since it refers to the presence of ASCII NUL bytes.  Also, matches the usage in the underlying CString API.